### PR TITLE
BRITECHSRM-325 Give speakers and admins own subobjects when return sessions

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

File: Sessions/SessionsRepository.fs [hunk 1]
```diff
@@ -16,6 +16,30 @@ open System.Data.SqlClient
 let connectionString = ConfigurationManager.ConnectionStrings.Item("DefaultConnection").ConnectionString
 let getConnection() = new MySqlConnection(connectionString)
 
+let emptyGuid = new Guid("00000000000000000000000000000000")
```
Comment (jebbinSCL) [inline Sessions/SessionsRepository.fs]: This can be replaced with `Guid.Empty`.

**Comment by lclarksl** [comment]
Amended, thanks @jebbinSCL

File: Sessions/SessionsRepository.fs [hunk 2]
```diff
@@ -16,6 +16,28 @@ open System.Data.SqlClient
 let connectionString = ConfigurationManager.ConnectionStrings.Item("DefaultConnection").ConnectionString
 let getConnection() = new MySqlConnection(connectionString)
 
+let entityToSession (entity : SessionSummaryEntity) : Session =
+    let speaker =
+        { Id = entity.SpeakerId
+          Forename = entity.SpeakerForename
+          Surname = entity.SpeakerSurname
+          Rating = enum<Rating> entity.SpeakerRating
```
Comment (jebbinSCL) [inline Sessions/SessionsRepository.fs]: This can just be `enum entity.SpeakerRating` I think. It will infer the type

**Comment by jebbinSCL** [comment]
:shipit: after tiny changes

File: Sessions/SessionsRepository.fs (hunk 2 above)
Comment (lclarksl) [inline Sessions/SessionsRepository.fs]: I tried and it doesn't

**Comment by ianlovell** [comment]
:shipit:

**Pull request merged**